### PR TITLE
docs: add blog `Apache APISIX helps Enterprises Transform Digitally`.

### PR DESCRIPTION
**Pull request by membphis**

### CODE REVIEW

**Comment by membphis** [comment]
@juzhiyuan @guoqqqi please take a look at this PR when you have time

**Comment by juzhiyuan** [comment]
OK

File: _posts/blog/zh-CN/apache-apisix-helps-enterprises-transform-digitally.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+| **swagger 格式 API 导入** | 使用 [swagger](https://swagger.io) 标准完成 API 批量导入                 | 已内置支持 [OpenAPI Specification3.0](https://swagger.io/specification) 方式导入 [swagger](https://swagger.io) 。|
+
+## 关于 Apache APISIX
+
+**1.Apache 顶级项目**
+
+APISIX 作为 Apache 顶级项目，中国**最快**毕业的顶级项目，Apache 唯一的 API 网关产品，也被纳入 CNCF 全景图。
+
+![1.jpg](https://static.apiseven.com/logo/20210203/1.png)
+
```
Comment (guoqqqi) [inline _posts/blog/zh-CN/apache-apisix-helps-enterprises-transform-digitally.md:41]: ``

File: _posts/blog/zh-CN/apache-apisix-helps-enterprises-transform-digitally.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+| **API 编排**              | 通过一个请求来调用多个微服务，并对返回结果做数据处理，最终整合成一个结果 | 支持两种方式完成此特性：<br>1. 通过 [GraphQL](https://graphql.org) 方式直接完成 API 的编排，目前 APISIX 已经开源支持。<br>2. 通过自定义插件方式扩充满足企业 API 编排需求。                                                                                                                                                                                                                                                |
+| **swagger 格式 API 导入** | 使用 [swagger](https://swagger.io) 标准完成 API 批量导入                 | 已内置支持 [OpenAPI Specification3.0](https://swagger.io/specification) 方式导入 [swagger](https://swagger.io) 。|
+
+## 关于 Apache APISIX
+
+**1.Apache 顶级项目**
+
+APISIX 作为 Apache 顶级项目，中国**最快**毕业的顶级项目，Apache 唯一的 API 网关产品，也被纳入 CNCF 全景图。
+
+![1.jpg](https://static.apiseven.com/logo/20210203/1.png)
```
Comment (juzhiyuan) [inline _posts/blog/zh-CN/apache-apisix-helps-enterprises-transform-digitally.md:40]:
```suggestion
![1.png](https://static.apiseven.com/logo/20210203/1.png)
```

**Pull request closed without merging**